cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaClusterCreator.java

```diff
@@ -180,7 +181,11 @@ private Future<KafkaCluster> brokerRemovalCheck(Kafka kafkaCr, KafkaCluster kafk
             usedToBeBrokersCheckFailed = false;
             return Future.succeededFuture(kafka);
         } else {
-            return brokerScaleDownOperations.brokersInUse(reconciliation, vertx, secretOperator, adminClientProvider)
+            return Future.join(
+                        ReconcilerUtils.clusterCaPemTrustSet(reconciliation, secretOperator),
+                        ReconcilerUtils.coPemAuthIdentity(reconciliation, secretOperator)
+                    )
+                    .compose(res -> brokerScaleDownOperations.brokersInUse(reconciliation, vertx, new TlsPemIdentity(res.resultAt(0), res.resultAt(1)), adminClientProvider))
```

Comment:
Would it make sense to have this in ReconcilerUtils as a utility method to provide directly the `TlsPemIdentity`? You have this on multiple places.

Reply:
Yeah I've moved the Pem ones to a ReconcilerUtils method, I've left the PKCS12 rather than have a record of records